core/src/main/resources/hudson/PluginManager/sidepanel.jelly

```diff
@@ -0,0 +1,117 @@
+<!--
```

Comment:
Definitely 👍 If it's alright with you though I'd like to wait a bit to have a more concrete design across Jenkins before building a new component/tag for these new sorts of sidebar behaviours as they may change/be tweaked still depending on future PRs.

Reply:
I am not sure if this is a good idea. As @daniel-beck noted as well:

> This is the second PR introducing a title in the sidepanel (after https://github.com/jenkinsci/jenkins/pull/6485) without even making a note of that. Is that documented and discussed somewhere? How are we going to handle the problem of pages without a sidepanel, won't it be weird when some pages have a title above the main panel, while others have it in the sidepanel?

I think we should start very soon with creating and documenting a control for that (and maybe opening a discussion about the new concept can and should be used in plugins). Or when do you expect that others will pick up those changes for their plugins? Then we should at least notify everybody that this concept is not yet mature and should not be applied in plugins.

Reply:
As far as I can tell #7051 is the codification of this design pattern. Am I correct in understanding that after #7051 is merged, this PR and the Configure Project redesign will be refactored to use the new codified design pattern, at which point this feedback will be completely addressed?

Reply:
The configure project is updated as part of #7051. The plugin manager on the other hand has been changed (I should probably update the opening post) to get sign off from Daniel so the title of the page is now above the search bar.  This could change in the future in which case I'll definitely adopt the way #7051 does it 👍

Reply:
I didn't really follow the explanation regarding the relationship between this PR and #7051, but I trust that we are making progress. My primary concern is that the design pattern codified in #7051 is consistently followed by all implementations, including the configure project page and the plugin manager page (this PR) as well as any future pages that are converted to this pattern.

The order of operations seems like it is up for debate, but my main concern is just that we reach the desired end state and not that we take any particular order of operations to get there. I would expect that it would be desirable to fully flesh out the API in #7051 before adding new implementations as in this PR, because otherwise those implementations would then need to be adjusted later to conform to the final API. But I could be wrong about this. Either way, I think we should get the API finalized and all implementations conforming before we convert more pages to this paradigm.